molecule/default/molecule-no-systemd.yml

```
@@ -17,13 +17,9 @@ lint: |
   flake8
 platforms:
   # Amazon Linux does not appear to offer Xfce
```

Might as well give our future selves a helpful pointer:
```suggestion
  # Amazon Linux 2023 does not currently offer Xfce
  # https://docs.aws.amazon.com/linux/al2023/release-notes/all-packages-al2023-20230419.html
```